Templates/TodoList-NodeJS/function.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "table",
```

Comment:
The problem here is that these input bindings are all eagerly evaluated before the method is invoked. So even for a POST, we'd be executing the query to get all the items. I don't think we want to do that. I think for something like this you'd want to do an IBinder style binding at runtime conditionally. We have a work item for enabling that in Node, but it isn't there yet.